spec/lib/vet360/contact_information/service_spec.rb

```diff
@@ -93,6 +93,34 @@
     end
   end
 
+  describe '#get_telephone_transaction_status' do
+    let(:transaction_id) { 'd47b3d96-9ddd-42be-ac57-8e564aa38029' }
+    let(:transaction) { Vet360::Models::Transaction.new(id: transaction_id) }
+
+    context 'when successful' do
+      it 'returns a status of 200' do
+        VCR.use_cassette('vet360/contact_information/telephone_transaction_status', VCR::MATCH_EVERYTHING) do
+          response = subject.get_telephone_transaction_status(transaction)
+          expect(response).to be_ok
+          expect(response.transaction).to be_a(Vet360::Models::Transaction)
+        end
+      end
+    end
+
+    context 'when not successful' do
+      it 'returns a status of 404' do
+        VCR.use_cassette('vet360/contact_information/telephone_transaction_status_error', VCR::MATCH_EVERYTHING) do
+          expect { subject.get_telephone_transaction_status(transaction) }.to raise_error do |e|
+            expect(e).to be_a(Common::Exceptions::BackendServiceException)
+            expect(e.status_code).to eq(404)
+            expect(e.errors.first.code).to eq('VET360_CORE103')
+            expect(e.errors.first.title).to eq('Not Found')
```

Comment:
Also not particularly concerned, but fwiw I've been resisting the urge to write assertions on the titles, etc out of fear of making the test too brittle without much value... feels like we want to make sure that the exception gets raised and that it's the right exception but I think we get that from asserting which code we expect. Just a thought.